pkg/xds/envoy/listeners/v3/static_endpoints_configurer.go

```diff
@@ -28,14 +29,26 @@ func (c *StaticEndpointsConfigurer) Configure(filterChain *envoy_listener.Filter
 				},
 			},
 			Name: envoy_common.AnonymousResource,
-			Action: &envoy_route.Route_Route{
+		}
+
+		if p.ClusterName != "" {
+			route.Action = &envoy_route.Route_Route{
 				Route: &envoy_route.RouteAction{
 					ClusterSpecifier: &envoy_route.RouteAction_Cluster{
 						Cluster: p.ClusterName,
 					},
 					PrefixRewrite: p.RewritePath,
 				},
-			},
+			}
+		} else if p.DirectResponse != nil {
```

Comment:
We have an existing `DirectResponseConfigurer`, but that one only receives `DirectResponseEndpoints`, and we need `StaticEndpointPath` along with DirectResponse in `admin_proxy_generator.go`. 
So I just added DirectResponse support into this `StaticEndpointsConfigurer`. I think we can also consider redirect the existing `DirectResponseConfigurer` to this `StaticEndpointsConfigurer`, and still keep the API of `envoy_listeners.DirectResponse` to enjoy its intuitiveness.